### PR TITLE
Docker folder

### PR DESCRIPTION
**Pull request by pgraverdy**
Add BO container start, nginx configuration

### CODE REVIEW

File: docker/configs/nginx/app.conf [hunk 1]
```diff
@@ -2,20 +2,34 @@
 # Nginx configuration for docker-compose nginx container
 #
 
-upstream tornadoserver {
+upstream aap-server {
```
Comment (rprimet) [inline docker/configs/nginx/app.conf:5]: app-server ?

File: docker/configs/nginx/app.conf [hunk 2]
```diff
@@ -2,20 +2,34 @@
...
+        proxy_pass http://backoffice-server;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+
     location / {
         proxy_set_header Host $host;
         proxy_set_header X-Real-Ip $remote_addr;
-        proxy_pass http://tornadoserver;
+        proxy_pass http://aap-server;
```
Comment (rprimet) [inline docker/configs/nginx/app.conf:30]: `app-server`

File: docker/configs/nginx/app.conf.dev [hunk 3]
```diff
@@ -2,20 +2,34 @@
 # Nginx configuration for docker-compose nginx container
 #
 
-upstream tornadoserver {
+upstream aap-server {
```
Comment (rprimet) [inline docker/configs/nginx/app.conf.dev:5]: `app-server`

File: docker/docker-compose-core.yml [hunk 4]
```diff
@@ -83,6 +83,49 @@ services:
...
+    command: ./start_server_bo.sh
+    restart: unless-stopped
+    tty: true
+    ports:
+      - "8890:8890"
+    environment:
+      ENV_MODE : "$ENV_MODE"
+      SHEET_ID : "$SHEET_ID"
+      TOKEN_LOC : "/home/icubam/resources/token.pickle"
+      SMS_KEY : "$SMS_KEY"
```
Comment (rprimet) [inline docker/docker-compose-core.yml:102]: plus nécessaire

File: docker/docker-compose-core.yml [hunk 5]
```diff
@@ -83,6 +83,49 @@ services:
...
+      SECRET_COOKIE : "$SECRET_COOKIE"
+      JWT_SECRET : "$JWT_SECRET"
+      GOOGLE_API_KEY : "$GOOGLE_API_KEY"
+      MB_KEY : "$MB_KEY"
```
Comment (rprimet) [inline docker/docker-compose-core.yml:106]: `MB_KEY` may be removed

File: docker/docker-compose-core.yml [hunk 6]
```diff
@@ -83,6 +83,49 @@ services:
...
+      NX_KEY : "$NX_KEY"
```
Comment (rprimet) [inline docker/docker-compose-core.yml:107]: `NX_KEY` may be removed

File: docker/docker-compose-core.yml [hunk 7]
```diff
@@ -83,6 +83,49 @@ services:
...
+      NX_API : "$NX_API"
```
Comment (rprimet) [inline docker/docker-compose-core.yml:108]: `NX_API` may be removed

**Comment by pgraverdy** [comment]
@dulacarnold deprecated keys removed

**Pull request merged**